metaci/notification/views.py

```diff
@@ -77,6 +80,23 @@ def add_plan_notification(request):
     )
 
 
+@login_required
```

Comment:
should notifs be staff_member_required?

Reply:
I don't think so but it's a matter of opinion. In this case I followed the existing pattern. If we want to update it to staff only then I think that's a different PR.

Reply:
fine by me, but we really need to take a look at our permissions. i'm worried if we can just send email to anyone.